requirements/base.txt

```diff
@@ -128,6 +130,8 @@ gunicorn==20.1.0
     # via apache-superset
 hashids==1.3.1
     # via apache-superset
+hijri-converter==2.2.4
+    # via holidays
```

Comment:
same thing here?

Reply:
Yes, those are changes I'm getting locally even when on `master` branch after running `pip-compile-multi --no-upgrade`.

Reply:
Tests are failing even locally, I cannot get them to run successfully not even using `master`. So will see if using 1.11.0 solves anything.

<img width="1373" alt="Screen Shot 2022-09-22 at 18 22 56" src="https://user-images.githubusercontent.com/38889534/191859113-166fc332-1ced-4d58-babf-f5811e6856f5.png">
<img width="1373" alt="Screen Shot 2022-09-22 at 18 19 46" src="https://user-images.githubusercontent.com/38889534/191859121-6f365e40-a4c6-4f59-a318-ea16b8ad36bc.png">